cli/commands/pack.go

```diff
@@ -81,12 +81,16 @@ func parsePackageDeps(deps []string, depsFile string) (common.PackDependencies,
 	}
 
 	if depsFile == "" && len(deps) == 0 {
-		defaultPackeDepsFilePath := filepath.Join(ctx.Project.Path, defaultPackageDepsFile)
-		if _, err := os.Stat(defaultPackeDepsFilePath); err != nil {
-			return nil, fmt.Errorf("Failed to use default package dependencies file: %s", err)
+		defaultPackDepsFilePath := filepath.Join(ctx.Project.Path, defaultPackageDepsFile)
+		if _, err := os.Stat(defaultPackDepsFilePath); err != nil {
+			if os.IsNotExist(err) {
+				defaultPackDepsFilePath = ""
+			} else {
+				return nil, fmt.Errorf("Failed to use default package dependencies file: %s", err)
+			}
```

Comment:
There is no reason for an additional `if` level.
Moreover, you shouldn't rewrite `defaultPackDepsFilePath`:

```go
if _, err := os.Stat(defaultPackDepsFilePath); err == nil {
    depsFile = defaultPackDepsFilePath
} else if !os.IsNotExist(err) {
    return nil, fmt.Errorf("Failed to use default package dependencies file: %s", err)
}
```